code/game/objects/items/weapons/switchtool.dm

```diff
@@ -126,12 +126,15 @@
 
 	if(!stored_modules[module])
 		return
+	if(deployed)
+		return
```

Comment:
1. This function is returning boolean values, so it should use that.
2. This function was converted from not returning anything, so it is not obvious in the code unless you read all the return statements that something actually comes out.
3. We hate Celt.